test/test_other.py

```diff
@@ -13417,7 +13417,7 @@ def test_closure_safe(self, args):
   def test_post_link(self):
     err = self.run_process([EMCC, test_file('hello_world.c'), '--oformat=bare', '-o', 'bare.wasm'], stderr=PIPE).stderr
     self.assertContained('--oformat=bare/--post-link are experimental and subject to change', err)
-    err = self.run_process([EMCC, '--post-link', 'bare.wasm'], stderr=PIPE).stderr
+    err = self.run_process([EMCC, '--post-link', 'bare.wasm', '-s', 'RUNTIME_DEBUG'], stderr=PIPE).stderr
```

Comment:
Maybe add a comment here about the reason for the -s flag? Otherwise a future refactoring might remove it as unneeded.